src/app/app.component.ts

```diff
@@ -57,20 +60,44 @@ export class AppComponent implements OnInit {
   }
 
   // connectet libpis with mqtt broker - Please check if this connection is okay I'm not sure about this code.
-  initConnection() {
-    (window as Window as LuminatorWindow).luminator.pis.init(
-      this.mqttConfig,
-    );
+  initConnection(): void {
+    (window as Window as LuminatorWindow).luminator.pis.init(this.mqttConfig);
```

Comment:
This can be shortened even more https://dev.azure.com/ltg-dev/TDC-Europe%20Software/_git/mSETSuite.PresentationCreator.Frontend?path=/src/types/index.d.ts